shell/src/main/java/alluxio/cli/fs/command/MvCommand.java

```diff
@@ -52,6 +52,9 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
     String[] args = cl.getArgs();
     AlluxioURI srcPath = new AlluxioURI(args[0]);
     AlluxioURI dstPath = new AlluxioURI(args[1]);
+    if (mFileSystem.exists(dstPath) && mFileSystem.getStatus(dstPath).isFolder()) {
```

Comment:
I guess you propose to do this change to keep consistent with hdfs, but shall we move this logic to the alluxio master side?

Reply:
Yes, I want to keep ALLUXIO's `mv` behavior consistent with HDFS. Is there any benefit to moving this operation to the master side? Can you suggest how to modify it?